app/views/quotes/new.html.erb

```diff
@@ -17,24 +18,91 @@
                      </button>
                   </div>
                </div>
+               <hr>
                <form action="" id="SubmitQuote">
+                  <div><b>Step 2:</b> Select service area</div>
+                  <div class="hidden" id="step2"> <!-- Displays instructions and gif -->
+                     <p>Identify your driveway, front door, and any area that needs to be cleared of snow and salted</p>
+                     <div>
+                        <%= image_tag('SelectArea.gif', alt: 'Select area example animation') %>
+                     </div>
+                  </div>
                   <div class="hidden row" id="displayTotals">
-                     <div class="col-12 mt-4">
+                     <div class="col-12 mt-4"> <!-- Displays total square footage selected -->
                         <div class="card">
                            <div class="card-body text-center p-1">
                               <span id="calculatedArea" class="font-weight-bold font-size-35"></span> sq ft
                            </div>
                         </div>
                      </div>
-                     <div class="col-12 mt-3">
+                     <div class="col-12 mt-3"> <!-- Displays subtotal for selected area -->
                         <div class="card">
                            <div class="card-body text-center p-1">
                               <span class="font-weight-bold currency-symbol text-secondary">$</span>
-                              <span id="totalDue" class="font-weight-bold font-size-35"></span> CAD
+                              <span id="subTotalDue" class="font-weight-bold font-size-35">subTotal</span> CAD
                            </div>
                         </div>
                      </div>
+                     <hr>
+                     <div class="col-12 mt-3"> <!-- Displays hint to add more areas -->
+                        <div>Did you know ...
+                           <p>You can select more than one area. Add side paths, backstairs, patio, etc </p>
+                        </div>
+                     </div>
+                  </div>
+                  <hr>
+                  <div><b>Step 3:</b> When would you like the service? </div>
+                  <div class="hidden row" id="displayServiceExpedition">
                      <div class="col-12 mt-3">
+                        <fieldset class="form-group">
+                           <div class="row">
+                              <div class="col-sm-10">
+                                 <% @service_expedition_time.each {|i| %>
```

Comment:
`i` is not a good variable name. Very hard to search/replace. In very few instances (e.g. nested loops over integers with `i`, `j`, and `k`) is it warranted and it is generally better to have a more descriptive variable name. In this case `service_expedition_time` or `time`. Even with a loop over integers, I prefer to use `index` or `idx` for the searchability.

Reply:
For multi-line blocks, prefer `do ... end` over `{ ... }`

Reply:
done